docs/src/pages/theming.mdx

```diff
@@ -103,51 +103,109 @@ accepts an object with keys that correspond to elements of component. The
 interfaces for the `appearance` prop for `UploadButton` and `UploadDropzone`
 are:
 
-```ts
-type StyleField =
-  | string
-  | CSSProperties
-  | ((args: CallbackArguments) => string | CSSProperties);
-
-type UploadButtonProps = {
-  /* rest of props */
-  appearance?: {
-    container?: StyleField;
-    button?: StyleField;
-    allowedContent?: StyleField;
-  };
-};
-
-type UploadDropzoneProps = {
-  /* rest of props */
-  appearance?: {
-    container?: StyleField;
-    uploadIcon?: StyleField;
-    label?: StyleField;
-    allowedContent?: StyleField;
-    button?: StyleField;
-  };
-};
-```
+<Tabs items={['ReactJS', 'SolidJS']}>
```

Comment:
are these tabs necessary? the types should be the same with the exception of signal-based frameworks will use getters instead of direct attributes?

Reply:
Well, difference is small, but still a difference. I wanted to make it clear for the user. Solid is signal based and also have bit different typedefs in couple of places. The main difference is that it leverages getters just like you said. How would you handle this case?

Reply:
we could probably just have a callout in the beginning saying that for signal-based frameworks the attributes are getters, but maybe tabs are fine. Mostly just seems like a hassle to maintain the types if you need to change in many places.

CSSProperties vs JSX.CSSProperties I'd say is just fine to refer to as CSSProperties